source/analytics/data.html.md.erb

```diff
@@ -14,4 +14,4 @@
   <% end %>
 </ul>
 
-<p class="govuk-body">These pages do not currently contain information about what happens to the data next: it is collected by Google Tag Manager and sent to Google Analytics.</p>
+<p class="govuk-body">These pages do not currently contain information about what happens to the data next. It is collected by Google Tag Manager and sent to Google Analytics.</p>
```

Comment:
Quick question - what do you think of changing this to "This documentation does not currently contain information about what happens to the data after the schema has been built (where it is collected by Google Tag Manager and sent to Google Analytics)."?

This does come down to personal preference though so happy to leave it as it is if you prefer!